report/4.Design/HardwareTest.tex

```diff
@@ -0,0 +1,24 @@
+\section{Hardware testing}
```

Comment:
I think in general this section needs to be streamlined in terms of what tense it uses. Because I actually think that past tense is wrong.

Reply:
Also I know my comments are not as good as Daniels but I am in a creative rut, and can only detect things I don't like